float64[] next_state
float64 reward
bool done
float64 target_cost
float64 unilateral
float64 friction
float64 singularity
float64 joint_range
float64 joint_torques
float64 no_touchdown
float64 smoothness
bool apex
---
float64 ack